src/assets/i18n/en_US.json

```diff
@@ -530,7 +530,7 @@
   "approval.detailspage.approval.heading": "Approval Details",
   "approval.detailspage.approval_date.label": "Date of Approval",
   "approval.detailspage.approval_status.label": "Approval Status",
-  "approval.detailspage.approval_status.system_rejected.comment": "The order could not be placed because the requisition was invalid.",
+  "approval.detailspage.approval_status.system_rejected.comment": "The order could not be placed because the requisition was invalid. This is caused by the following reason(s):",
```

Comment:
Use num plural here like: {{ num, plural, =0{} =1{This is caused by the following reason:} other{This is caused by the following reasons:} }}
I hope it will work - I haven't checked it out

Pls change also the German and French translation or ask Mandy to do it.